Examples-Web/IdleTimeout/src/Hooks/IdleTimeout.tsx

```diff
@@ -1,36 +1,36 @@
-import React, { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 
-import { ConnectionManager } from 'TouchFree/src/Connection/ConnectionManager';
+import TouchFree, { EventHandle } from 'TouchFree/src/TouchFree';
 
-export type AppState = 'Idle' | 'Active';
-
-// Custom hook which will return app state based on hand presence.
+// Custom hook which will return whether the application is idle or not
+// based on hand presence.
 // If no hands have been observed for specified timeout in milliseconds
 // the state will be idle.
-// ConnectionManager.init() must be called before this hook.
-const useIdleTimeout = (timeoutMs: number): AppState => {
-    const [appState, setAppState] = React.useState<AppState>('Idle');
+// TouchFree.Init() must be called before this hook.
+const useIdleTimeout = (timeoutMs: number): boolean => {
+    const [isIdle, setIsIdle] = useState(true);
 
     useEffect(() => {
         let timeoutId: NodeJS.Timeout;
-        const timeoutFunction = () => setAppState('Idle');
+        const timeoutFunction = () => setIsIdle(true);
```

Comment:
🤷 This is only used once so don't think it needs it own function

Reply:
Removed!